pymeasure/instruments/nf/li5660.py

```diff
@@ -0,0 +1,445 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2020 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+from pymeasure.instruments import Instrument #, RangeException
+from pymeasure.instruments.validators import truncated_range, strict_discrete_set
+# from pymeasure.adapters import PrologixAdapter
+# from pymeasure.instruments.validators import truncated_range, strict_discrete_set
+
+# import numpy as np
+# import time
+# from io import BytesIO
+# import re
+
+class LI5660(Instrument):
+
+    def __init__(self, resourceName, **kwargs):
+        super().__init__(
+            resourceName,
+            "NF Lock-In Amplifier LI5600",
+            **kwargs
+        )
+    
+    amplifier_id = Instrument.measurement(
```

Comment:
the standard SCPI query "*IDN?" is already defined in the `Instrument` base class (`Instrument.id`).

Reply:
This line removed